src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -154,4 +166,104 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
             }
         }));
     }
+
+    private async Task SubscribeDashboardLogsAsync()
+    {
+        string? dashboardResourceId = null;
```

Comment:
Move resolving the dashboard resource ID to its own method

Reply:
Done